packages/ui/src/components/DataStory/DataStory.tsx

```diff
@@ -3,24 +3,8 @@ import { Workbench } from './Workbench';
 import { DataStoryProvider } from './store/store';
 import { DataStoryProps } from './types';
 
-export const DataStory =  ({
-  server,
-  initDiagram,
-  callback,
-  hideToolbar = false,
-  slotComponents,
-  observers,
-  onInitialize,
-}: DataStoryProps) => {
+export const DataStory = (props : DataStoryProps) => {
   return <DataStoryProvider>
-    <Workbench
-      server={ server }
-      initDiagram={ initDiagram }
-      callback={ callback }
```

Comment:
Drive by comment: "callback" is a horrible naming, we should really tell what that "callback" does 😅